src/main/java/org/shipkit/auto/version/VersionConfig.java

```diff
@@ -52,19 +56,25 @@ String newPatchVersion() {
      */
     static VersionConfig parseVersionFile(File versionFile) {
         Properties p = new Properties();
+
+        if (!versionFile.exists()) {
+            return new VersionConfig(null, "v");
+        }
+
         try {
             p.load(new FileReader(versionFile));
         } catch (IOException e) {
-            throw new ShipkitAutoVersionException(
-                    "Please create file 'version.properties' with a valid 'version' property, " +
-                            "for example 'version=1.0.*'", e);
+            System.out.println("[shipkit-auto-version] Ignoring file '" + versionFile.getName()
```

Comment:
Let's follow up with a little clean up for this, but let's not block this PR for this edge case.